App/src/main/java/cc/softwarefactory/lokki/android/MapViewFragment.java

```diff
@@ -138,6 +142,39 @@ public void onResume() { // onResume is called after onActivityCreated, when the
         }
     }
 
+    private void checkLocationServiceStatus() {
+        LocationManager lm = (LocationManager) context.getSystemService(Context.LOCATION_SERVICE);
+        boolean gps = false;
+        boolean network = false;
+        try{
+            gps = lm.isProviderEnabled(LocationManager.GPS_PROVIDER);
+        }catch(Exception ex){
+
+        }
+        try{
+            network = lm.isProviderEnabled(LocationManager.NETWORK_PROVIDER);
+        }catch(Exception ex){
+
```

Comment:
Empty catch blocks and no spaces after and before brackets. Also catch only the exception that the method is throwing rather than all exceptions, if possible.

In the catch blocks at least log an error message.